client/pages/login.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <AuthForm>
+    <InputField class="w-full" field-name="Email" field-type="text" />
+    <InputField class="w-full" field-name="Password" field-type="password" />
+    <a
+      href="/forgot-password"
+      class="self-end text-xs font-semibold text-white underline underline-offset-2"
+    >
+      forgot password?
+    </a>
+    <button
+      class="place-self-center text-l font-bold text-red bg-white w-24 h-8 rounded border border-solid border-red my-5"
+    >
+      Login
+    </button>
+    <p class="place-self-center">
+      don't have an account?
+      <a
+        href="sign-up"
+        class="text-blue font-semibold underline underline-offset-2"
+      >
+        sign up!
+      </a>
+    </p>
+  </AuthForm>
+</template>
+
+<script>
+import AuthForm from '~/components/Auth/AuthForm.vue';
```

Comment:
These don't need to be imported if the directory containing the components are defined in the nuxt configs